hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterAdminCLI.java

```diff
@@ -1925,6 +1924,43 @@ public void testAddMultipleMountPointsFailure() throws Exception {
         err.toString().contains("Cannot add mount points: [/testAddMultiMountPoints-01"));
   }
 
+  @Test
+  public void testMountPointWithDifferentDestinations() throws Exception {
+    System.setOut(new PrintStream(out));
+    System.setErr(new PrintStream(err));
+
+    stateStore.loadCache(MountTableStoreImpl.class, true);
+    String[] argv = new String[] {"-add", "/multipleDestination0", "ns01,ns02", "/tmp0"};
```

Comment:
check if ns01, ns02 works or not, else while getting the ns we should trim as well in the prod code